test/unit/cssnano-simple/cssnano-preset-simple/plugin.ts

```diff
@@ -12,6 +12,7 @@ const cssnanoPlugin = (options = {}) => {
   for (const nanoPlugin of nanoPlugins) {
     if (Array.isArray(nanoPlugin)) {
       let [processor, opts] = nanoPlugin
+      // @ts-expect-error Property 'default' does not exist on type 'PluginCreator<any>'.ts(2339)
       processor = processor.default || processor
```

Comment:
I guess this error means we don't need `.default` anymore?

Reply:
Yeah, the related test passes, so removed it!